handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -34,7 +34,6 @@
 
 import java.nio.ByteBuffer;
 import java.nio.ReadOnlyBufferException;
-import java.security.AlgorithmConstraints;
```

Comment:
Yah it looks like we don't want this and it was introduced recently [1]. We store a reference to an algorithm constraint as an `Object algorithmConstraints` to avoid referencing this JDK 1.7+ class I guess.

[1] https://github.com/netty/netty/commit/325cc84a2ebbf4f5e362ec9ca09202b4647bd100#diff-cfd417915faad637e4278b54adafd9baR37